lib/foreman_messager.rb

```diff
@@ -5,12 +5,8 @@ def initialize(bot)
     @bot = bot
   end
 
-  def update_team_id(team_id)
-    @team_id = team_id
-  end
-
   def send(message)
-    @bot.send(message, @team_id, foreman_id)
```

Comment:
`Bot#send` takes three arguments, which confirms my suspicion that this never gets used. If it was, we'd regularly see `ArgumentError` popping up in production.